packages/cli/src/commands/source/push/add.ts

```diff
@@ -95,13 +96,15 @@ export default class SourcePushAdd extends Command {
     await Promise.all(
       flags.folder.flatMap((folder) => {
         return Promise.all(
-          readdirSync(folder).flatMap(async (file) => {
-            const fullPath = path.join(folder, file);
-            const docBuilders =
-              parseAndGetDocumentBuilderFromJSONDocument(fullPath);
-            this.successMessageOnParseFile(fullPath, docBuilders.length);
-            await send(docBuilders);
-          })
+          readdirSync(folder)
+            .filter((file) => !isDotFile(file))
```

Comment:
Instead of `.filter((file) => !isDotFile(file))`, how about `.filter((file) => isJsonFile(file))` ?

Reply:
I though it would be useful if the user had saved JSON data using a different filetype extension (e.g. JSON in a txt file)...

I don't know if it will happen and if it might create confusion... Anyhow, we can simply accept `.json` files.

Reply:
I would be tempted to say that since the whole command description is about pushing JSON file... it's fair to say that it's a no brainer that only `.json` file would be processed, and that it should not cause confusion.